### PR TITLE
Math magicians -  events

### PR DESCRIPTION
**Pull request by codecaiine**
-  Created a new feature branch
-  Implemented the calculator logic 
- Used the utility functions

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,254 +1,122 @@
...
+    render() {
+      return (
+        <div className="calculator">
+          <div className="output">
+            <p>
+              {' '}
+              { this.formatOutput() }
+              {' '}
+
+            </p>
```
Comment (george-shammar) [inline src/components/Calculator.js:96]: - [ ] Please, handle the division by zero error. When I divided a number by zero, I got this browser error `Error: [big.js] Division by zero`. Please, let's display an error message on the calculator screen instead to tell users that they can't divide by zero.
- [ ] The same thing is applicable to when I mistakenly double-clicked an operator, I received this browser error, `TypeError: one.times is not a function`. Please, let's fix it.
- [ ] The Licence link in your README is broken as well. Please, let's make our README professional at all times.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @codecaiine , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-shammar — CHANGES_REQUESTED** [review]
@codecaiine, Great job.

File: src/components/Calculator.js (hunk 1 above)
Comment (codecaiine) [reply src/components/Calculator.js:96]: Hey @george-shammar,

Thanks for reviewing my work.
I implemented and fixed all requirements.

Thanks.

**Review by henatan99 — APPROVED** [review]
Hi @codecaiine , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**